internal/security/gnxi/rpcexec.go

```diff
@@ -505,10 +513,17 @@ func GnsiAuthzRotate(ctx context.Context, dut *ondatra.DUTDevice, opts []grpc.Di
 	if err != nil {
 		return err
 	}
-	_, err = gnsiCStream.Recv()
-	// invalid policy is expected since the empty policy is not allowed
-	if strings.Contains(err.Error(), "invalid policy") || status.Code(err) == codes.InvalidArgument {
-		return nil
+	for {
+		_, err := gnsiCStream.Recv()
+		if err != nil {
+			if strings.Contains(err.Error(), "invalid policy") || status.Code(err) == codes.InvalidArgument || strings.Contains(err.Error(), "InvalidArgument") {
+				return nil
+			} else {
```

Comment:
why else?

Reply:
When Authz rotate fails we make sure that the failure is for expected ones to return NIL code which is required for negative validations in feature/security/gnsi/authz/tests/authz/authz1_4_test.go.  In else section we are returning error code which may be genuine rotate failure.

Reply:
here too we should skip the else. as above.

Reply:
Fixed this. 

Thanks